src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/UploadTranslatedDocumentHandler.java

```diff
@@ -22,11 +23,12 @@ public class UploadTranslatedDocumentHandler extends CallbackHandler {
     private static final List<CaseEvent> EVENTS = List.of(UPLOAD_TRANSLATED_DOCUMENT);
     private final UploadTranslatedDocumentStrategyFactory uploadTranslatedDocumentStrategyFactory;
 
-    private final Map<String, Callback> callbackMap = Map.of(callbackKey(ABOUT_TO_SUBMIT), this::uploadDocument);
-
     @Override
     protected Map<String, Callback> callbacks() {
-        return callbackMap;
+        return Map.of(
```

Comment:
perhaps it is better when the map is created only one during the bean creation rather than every time we call the method. thats how it was before in this handler?

Reply:
done

Reply:
regarding submitting the claimant reply.. Those changes will be added in Kadir's PR as his ticket has dependency on that.